### ISSUE TITLE
Test with training data

### ISSUE THREAD

**Issue by tensor-flower** [MaybeShewill-CV/CRNN_Tensorflow#172]
For some reason, whenever I'm testing my model, it never uses images in my test folder, it always uses part of my training data. Any idea why this is happening?

I generated ```train_feature.tfrecords, val_feature.tfrecords, test_feature.tfrecords``` all correctly, and I had the respective ```sample.txt``` for train and test correctly.

I have 1400 training images, 15% of them (210) are validation, and 50 are test set. When I test, it's weird that I'm testing 50 images, but none of the 50 are from my test set.

**Comment by tensor-flower** [comment]
Issue can be resolved from changing line 216 in local_utils/data_utils.py, change from ```tfrecords_path = os.path.join(cfg.PATH.TFRECORDS_DIR, "train_feature.tfrecords")``` to ```tfrecords_path = os.path.join(cfg.PATH.TFRECORDS_DIR, "test_feature.tfrecords")```

Of course this solves the issue at test time, need to change it back at train time. Maybe add an if clause? I can do a pull request.
